umcontroller/umcontroller.go

```diff
@@ -400,6 +403,14 @@ func (umCtrl *Controller) UpdateComponents(
 	return umCtrl.currentComponents, umCtrl.updateError
 }
 
+func (umCtrl *Controller) SubscribeNewComponentListener() <-chan cloudprotocol.ComponentStatus {
```

Comment:
Currently, we have only one consumer for new components and basically, one channel can be returned from resource usage point of view.